src/services/votes.ts

```diff
@@ -9,130 +8,42 @@ import { CycleStatusType } from '../types/cycles';
 import { z } from 'zod';
 import { quadraticVoting } from '../modules/quadraticVoting';
 
-/**
- * Handler to receive the votes for a specific cycle and user.
- * @param {PostgresJsDatabase<typeof db>} dbPool - The database connection pool.
- * @param {Request} req - The Express request object.
- * @param {Response} res - The Express response object.
- */
-export function getVotes(dbPool: PostgresJsDatabase<typeof db>) {
-  return async function (req: Request, res: Response) {
-    const userId = req.session.userId;
-    const cycleId = req.params.cycleId;
-
-    if (!cycleId) {
-      return res.status(400).json({
-        errors: [
-          {
-            message: 'Expected cycleId in query params',
-          },
-        ],
-      });
-    }
-
-    const votesRow = await getVotesForCycleByUser(dbPool, userId, cycleId);
-
-    return res.json({ data: votesRow });
-  };
-}
-
-/**
- * Retrieves the votes for a specific cycle and user.
- * @param {PostgresJsDatabase<typeof db>} dbPool - The database connection pool.
- * @param {string} userId - The ID of the user.
- * @param {string} cycleId - The ID of the cycle.
- */
-export async function getVotesForCycleByUser(
-  dbPool: PostgresJsDatabase<typeof db>,
-  userId: string,
-  cycleId: string,
-) {
-  const response = await dbPool.query.cycles.findMany({
-    with: {
-      forumQuestions: {
-        with: {
-          questionOptions: {
-            with: {
-              votes: {
-                where: ({ optionId }) =>
-                  sql`${db.votes.createdAt} = (
-                    SELECT MAX(created_at) FROM (
-                        SELECT created_at, user_id FROM votes 
-                        WHERE user_id = ${userId} AND option_id = ${optionId}
-                    ) as ranked
-                  )`,
-              },
-            },
-          },
-        },
-      },
-    },
-    where: eq(db.cycles.id, cycleId),
-  });
-
-  const out = response.flatMap((cycle) =>
-    cycle.forumQuestions.flatMap((question) =>
-      question.questionOptions.flatMap((option) => option.votes),
-    ),
-  );
-
-  return out;
-}
-
 /**
  * Handler function that saves votes submitted by a user.
  * @param {PostgresJsDatabase<typeof db>} dbPool - The database connection pool.
  * @param {Request} req - The Express request object containing the user's submitted votes.
  * @param {Response} res - The Express response object to send the result.
  */
```

Comment:
Adjust ts-docstring because it is not a handler function anymore containing a request and response parameter.